calliope/backend/pyomo/constraints/energy_balance.py

```diff
@@ -83,6 +83,12 @@ def load_constraints(backend_model):
             backend_model.timesteps,
             rule=balance_storage_constraint_rule
         )
+    if 'loc_techs_storage_discharge_depth' in sets:
```

Comment:
As mentioned before, I'd move this to 'dispatch.py', to fit with the placement of `storage_max_constraint_rule`

Reply:
Done